File: src/main/kotlin/net/wavem/uvc/rms/gateway/event/domain/Event.kt
package net.wavem.uvc.rms.gateway.event.domain

import net.wavem.uvc.rms.common.domain.header.Header
import net.wavem.uvc.rms.gateway.event.domain.com_info.ComInfo
import net.wavem.uvc.rms.gateway.event.domain.event_info.EventInfo
import net.wavem.uvc.rms.gateway.event.domain.job.EventTaskInfo

data class Event(
    val header : Header?,
    val taskInfo : EventTaskInfo?,
    val eventInfo : EventInfo?,
    val comInfo : ComInfo?
) {
}

File: src/rms/response/event/domain/event.py

class Event():
    def __init__(
        self,
        header: dict = None,
        taskInfo: dict = None,
        eventInfo: dict = None,
        comInfo: dict = None
    ) -> None:
        self.header = header
        self.taskInfo = taskInfo
        self.eventInfo = eventInfo
        self.comInfo = comInfo
        

__all__ = ['event']